modules/telemetry/telemetry_utils.py

```diff
@@ -144,6 +144,9 @@ def __init__(
 
     def run(self):
         while True:
+            # Sleep for 1 ms
+            time.sleep(0.001)
```

Comment:
Moderate problem; it seems the import from time is `from time import time`, so `time.sleep()` won't work here (why Pyright is failing).

Could you either add to the import statement to do `from time import time, sleep` or change the statement to `import time` and replace all other occurrences of `time()` with `time.time()`?

Reply:
Oops, not sure how I was running locally (I had a `import time` that I forgot to
stage). Fixed!